index.html

```diff
@@ -14,6 +16,61 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About</a>
+      <a href="#">Portfolio</a>
+      <a href="#">Contact</a>
+    </nav>
+
+
+    <header>
```

Comment:
Although not required, it's best practice to contain the navigation within the header.